openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIResponseMapper.java

```diff
@@ -516,8 +515,9 @@ private void setCacheConfigField(CacheConfigAnnotation cacheConfig, String field
                 apiResponse = new ApiResponse();
                 apiResponse.description("Any Response");
                 Content content = new Content();
-                content.put(WILD_CARD_CONTENT_KEY, new io.swagger.v3.oas.models.media.MediaType().example(new Example()
-                        .summary(WILD_CARD_SUMMARY)));
+                io.swagger.v3.oas.models.media.MediaType mediaType = new io.swagger.v3.oas.models.media.MediaType();
+                mediaType.setSchema(new Schema<>().description(WILD_CARD_SUMMARY));
```

Comment:
Cant we avoid using the full qualified class names in here?

Reply:
When we are going to use `MediaType` there is another MediaType type that comes from java packages 
`javax.ws.rs.core.MediaType` . Therefore if we remove the full qualified class names if made an error since their implementation is different from each other.